N/NLPModelsJuMP/Compat.toml

```diff
@@ -26,13 +26,12 @@ NLPModels = "0.18-0.19"
 MathOptInterface = "1.7.0-1"
 
 ["0.12-0.12.1"]
-JuMP = "1.2.0-1"
+JuMP = "1.2-1.14.1"
 
 ["0.12.1-0"]
 NLPModels = "0.18-0.20"
 
 ["0.12.2-0"]
-JuMP = "1.2-1.14.1"
```

Comment:
This version doesn't have any compat at all with jump, or am I missing something?

Reply:
No, you're right, I forgot to change the version range in line 28

Reply:
Just informing, @odow pointed out that we actually don't want to add constraints because we are still working, we just don't support the new features yet.